docs/configure/triggers.md

```diff
@@ -41,7 +41,7 @@ Whenever data of the specified data types is ingested, a `POST` request will be
 
 {{% alert title="Note" color="note" %}}
 
-You must have [data capture](/services/data/capture/) and [cloud sync](/services/data/cloud-sync/) configured for the relevant components to use this trigger and the component must return the type of data you configure in the trigger's **Data Types**.
+You must have [data capture and cloud sync](/services/data/capture-sync/) configured for the relevant components to use this trigger and the component must return the type of data you configure in the trigger's **Data Types**.
```

Comment:
"You must have [data capture and cloud sync](https://github.com/services/data/capture-sync/) configured for the relevant components to use this trigger" i dont think this is explicitly true either; e.g., Tennibot is using triggers right now in prod and i am under the impression that they do not have data capture turned on (they're only using cloud sync in conjunction w/ their own data writing logic; not viam data capture).

Reply:
I think let's just remove this note @JessamyT

Reply:
```suggestion
```